frontend/src/app/modules/workspace/video/video-page/video-page.component.html

```diff
@@ -1,16 +1,33 @@
 <div class="w-100">
   <div class="actions-container">
     <button mat-button class="button-outlined">{{ viewsNumber }} views</button>
-    <button mat-button class="button-accent">Share</button>
-    <button mat-icon-button>
-      <mat-icon svgIcon="details"></mat-icon>
+    <button mat-button class="button-accent">
+      Share
     </button>
     <button mat-icon-button>
-      <mat-icon svgIcon="share-link"></mat-icon>
+      <mat-icon svgIcon="details"></mat-icon>
     </button>
   </div>
   <app-video-player></app-video-player>
-  <app-video-reactions class="reactions-container"></app-video-reactions>
+  <div *ngIf="isCurrentVideo()">
```

Comment:
```suggestion
  <div *ngIf="currentVideo && currentUser">
```

Reply:
But this won't work until you add getting of the current user in this component.
See, you have this property here and pass it to the child, but never initialize. 
Missing `currentUser` is a reason for that missing `avatarLink` error